scm-ui/ui-webapp/src/repos/sources/modules/sources.test.ts

```diff
@@ -166,7 +191,7 @@ describe("sources fetch", () => {
       const actions = store.getActions();
       expect(actions[0].type).toBe(FETCH_SOURCES_PENDING);
       expect(actions[1].type).toBe(FETCH_SOURCES_FAILURE);
-      expect(actions[1].itemId).toBe("scm/core/_/");
+      expect(actions[1].itemId).toBe("scm/core/_//");
```

Comment:
i don't get this. why are you expecting two ending slashes?

Reply:
I've appended the hunk with a slash. This is the root path without an explicit hunk